### DISCUSSION TITLE
Welcome, Abdul Latheef Faiq Ahamed!

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#173] [Subscribe]
Ahoy @Faiqlatheef, thank you for joining us! 😊

**Comment by thuongtruong1009** [comment]
Hi @Faiqlatheef , welcome to Yolo!

**Comment by Faiqlatheef** [comment]
Thank you so much @ParanoidUser @thuongtruong1009 
can you guys pls help me to get starstruck badge?

**Reply by nguyen-tuan-hiep** [reply]
give a star on my repo and I will star you back: https://github.com/nguyen-tuan-hiep/Introduction-to-Communication-Engineering
